test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/ViewModels/ReadMePreviewViewModelTests.cs

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.Sdk.TestFramework;
+using Moq;
+using NuGet.PackageManagement.UI.ViewModels;
+using Xunit;
+
+namespace NuGet.PackageManagement.UI.Test.ViewModels
+{
+    [Collection(MockedVS.Collection)]
+    public class ReadMePreviewViewModelTests
+    {
+        //[Fact]
```

Comment:
I'm guessing these will be reenabled eventually.

Reply:
Yea! With all the flux that happened with implementation details I think maybe doing a separate pr just for unit tests might be good.

Reply:
Are you planning to enable these tests?